### DISCUSSION TITLE
PDF preview on Windows

### DISCUSSION THREAD

**Discussion by tayopi** [sxyazi/yazi#1831] [1. Q&A]
I cannot seem to get pdf files to preview on Windows.

I'm using PowerShell v7.4.5 in Windows Terminal v1.21.2911.0. 

Here are the steps I've taken so far.
1. I've used scoop to install poppler. 
2. I have the following lines to my `yazi.toml` file:

```toml
[plugin]
...
preloaders = [
   ...
   # PDF
   { mime = "application/pdf", run = "pdf" },
   ...
]
previewers = [
   ...
   # PDF
   { mime = "application/pdf", run = "pdf" },
   ...
]
```

3. And I have added the `pdf.lua` file to `~AppData\Roaming\yazi\config\plugins`

After all of that, if I hover over a pdf file in yazi, all I get is an empty preview window. What am I doing wrong?

**Comment by sxyazi** [comment]
Send your `yazi --debug` please

**Comment by tayopi** [comment]
Yazi
    Version: 0.3.3 (7c445ce 2024-09-04)
    Debug  : false
    OS     : windows-x86_64 (windows)

Ya
    Version: 0.3.3 (7c445ce 2024-09-04)

Emulator
    Emulator.via_env: ("", "")
    Emulator.via_csi: Ok(Unknown([]))
    Emulator.detect : Microsoft

Adapter
    Adapter.matches: Sixel

Desktop
    XDG_SESSION_TYPE           : None
    WAYLAND_DISPLAY            : None
    DISPLAY                    : None
    SWAYSOCK                   : None
    HYPRLAND_INSTANCE_SIGNATURE: None
    WAYFIRE_SOCKET             : None

SSH
    shared.in_ssh_connection: false

WSL
    WSL: false

Variables
    SHELL              : None
    EDITOR             : Some("micro")
    VISUAL             : None
    YAZI_FILE_ONE      : Some("C:\\Program Files\\Git\\usr\\bin\\file.exe")
    YAZI_CONFIG_HOME   : None

Text Opener
    default: Some(Opener { run: "micro %*", block: true, orphan: false, desc: "Micro", for_: None, spread: true })
    block  : Some(Opener { run: "micro %*", block: true, orphan: false, desc: "Micro", for_: None, spread: true })

Multiplexers
    TMUX               : false
    tmux version       : program not found
    ZELLIJ_SESSION_NAME: None
    Zellij version     : program not found

Dependencies
    file             : 5.45
    ueberzugpp       : program not found
    ffmpegthumbnailer: program not found
    magick           : 7.1.1-39
    fzf              : 0.55.0
    fd               : 10.2.0
    rg               : 14.1.1
    chafa            : 1.14.4
    zoxide           : 0.9.6
    7z               : 24.08
    7zz              : program not found
    jq               : 1.7.1

**Comment by sxyazi** [comment]
> in Windows Terminal v1.21.2911.0

Your terminal version is too low to support image display, upgrade your terminal to >= v1.22.2702.0, see https://yazi-rs.github.io/docs/image-preview#windows
